exercises/chapter10/src/Data/AddressBook.purs

```diff
@@ -21,19 +25,12 @@ data PhoneType
   | CellPhone
   | OtherPhone
 
-instance showPhoneType :: Show PhoneType where
-  show HomePhone = "HomePhone"
-  show WorkPhone = "WorkPhone"
-  show CellPhone = "CellPhone"
-  show OtherPhone = "OtherPhone"
-
-derive instance genericPhoneType :: Generic PhoneType _
-
-instance encodeJsonPhoneType :: EncodeJson PhoneType where
-  encodeJson = genericEncodeJson
-
-instance decodeJsonPhoneType :: DecodeJson PhoneType where
-  decodeJson = genericDecodeJson
+-- ANCHOR: PhoneType_generic
+derive instance genericPhoneType    :: Generic    PhoneType _
+instance        encodeJsonPhoneType :: EncodeJson PhoneType where encodeJson = genericEncodeJson
+instance        decodeJsonPhoneType :: DecodeJson PhoneType where decodeJson = genericDecodeJson
+-- ANCHOR_END: PhoneType_generic
+instance        showPhoneType       :: Show       PhoneType where show       = genericShow
```

Comment:
Wondering if `genericShow` should be used for `showPhoneType` in Ch7 and Ch8 too. Will investigate further after merging this PR and #240.